arcgis-ios-sdk-samples/Scenes/Get Elevation at a point/ElevationViewController.swift

```diff
@@ -0,0 +1,20 @@
+// Copyright 2019 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+
+class ElevationViewController: UIViewController {
+    @IBOutlet var labelView: UIView!
```

Comment:
This outlet is just the view controller's view, so it isn't necessary. Please remove it.